src/realm/sync/noinst/client_history_impl.hpp

```diff
@@ -457,7 +457,7 @@ class ClientReplication final : public SyncReplication {
     }
 
     // Overriding member functions in realm::Replication
-    version_type prepare_changeset(const char*, size_t, version_type) override final;
+    version_type prepare_changeset(const char*, size_t, version_type) final;
```

Comment:
since this class is declared final, i think we just need to mark this as override. and if we're gonna change this line we should change the one right below it too.